mpmath/__init__.py

```diff
@@ -19,6 +19,17 @@
 fp._iv = iv
 iv._iv = iv
 
+
+def __getattr__(name):
+    if name == "dps":
+        raise AttributeError(
+            f"Module {__name__!r} does not have attribute {name!r}; "
```

Comment:
This won't work for 3.5, but don't worry.

Reply:
Yeah forgot to say the PEP 562 needs Python 3.7